dependencies.yaml

```
@@ -27,10 +27,28 @@ files:
     includes:
       - checks
       - py_version
+  docs:
+    output: none
+    includes:
+      - cudatoolkit
+      - py_version
+      - docs_builds
 channels:
   - rapidsai
   - conda-forge
 dependencies:
+  docs_builds:
+    common:
+      - output_types: [conda, requirements]
```

Doxygen isn't available via PyPI, so we should remove `requirements`.
```suggestion
      - output_types: [conda]
```